packages/util-crypto/src/secp256k1/keypair/fromSeed.spec.ts

```diff
@@ -14,7 +14,7 @@ const tests: Test[] = [
     'life fee table ahead modify maximum dumb such tobacco boss dry nurse',
     '0xf2360e871c830d397fe221382b503f07ddd8763df81a94bb2504390a2fb91f59',
     '0x036b0aa6beab469dd2b748a0ff5ddbe3d13df1e15c9d28a2aa057212994e127bea',
-    '0xae8e8fcacbaeb607bcdf0bbd7e615f2b4ef484ee54f19d68a7393fb6db2dd9cd'
+    '0xae8e8fcacbaeb607bcdf0bbd7e615f2b4ef484ee54f19d68a7393fb6db2dd9cd' // why are those here when the test only used the first 3 of the array?
```

Comment:
```suggestion
    '0xae8e8fcacbaeb607bcdf0bbd7e615f2b4ef484ee54f19d68a7393fb6db2dd9cd'
```

Reply:
Not sure why the comment is here?

This is the converted seed, not checked since the priv/pub is enough.

Reply:
No I was just wondering if we should delete those.